sc/src/main/java/uk/ac/ebi/atlas/search/JsonGeneSearchController.java

```diff
@@ -158,6 +158,9 @@ public String search(@RequestParam MultiValueMap<String, String> requestParams)
                                                 experimentAccession,
                                                 geneId,
                                                 markerGeneFacets.get(geneId).get(experimentAccession)));
+                            } else {
+                                experimentAttributes.put(
+                                        "markerGenes", new ArrayList());
```

Comment:
In general it’s better to use [`Collections.emptyList()`](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptyList--) or Guava’s [`ImmutableList.of()`](https://google.github.io/guava/releases/24.1-jre/api/docs/com/google/common/collect/ImmutableList.html#of--) to make the emtpy list immutable and communicate your intent clearly, since we know that list can’t be modified.